frontend/src/metabase/browse/metrics/BrowseMetrics.tsx

```diff
@@ -86,15 +98,40 @@ export function BrowseMetrics() {
 }
 
 function MetricsEmptyState() {
+  const newMetricLink = Urls.newQuestion({
+    mode: "query",
+    cardType: "metric",
+  });
+
+  const { url: metricsDocsLink, showMetabaseLinks } = useDocsUrl(
+    "data-modeling/metrics",
+  );
+
   return (
     <Flex align="center" justify="center" mih="70vh">
       <Box maw="25rem">
         <EmptyState
           title={t`Metrics help you summarize and analyze your data effortlessly.`}
           message={
-            <Text mt="sm" maw="25rem">
-              {t`Metrics are like pre-defined calculations: create your aggregations once, save them as metrics, and use them whenever you need to analyze your data.`}
-            </Text>
+            <Box>
+              <Text mt="sm" maw="25rem">
+                {t`Metrics are like pre-defined calculations: create your aggregations once, save them as metrics, and use them whenever you need to analyze your data.`}
+              </Text>
+              <Flex pt="md" align="center" justify="center" gap="md">
+                {showMetabaseLinks && (
+                  <Link
+                    target="_blank"
+                    to={metricsDocsLink}
+                    variant="brandBold"
+                  >{t`Read the docs`}</Link>
+                )}
+                <Button
```

Comment:
This button should be rendered conditionally, just like in the "New" menu, see https://github.com/metabase/metabase/blob/fb4b8ca2a60fede819fff9362956941ccccadbcb/frontend/src/metabase/components/NewItemMenu/NewItemMenu.tsx#L122